frontend/src/index.html

```diff
@@ -242,95 +242,119 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
     </div>
 
     <!-- player UI design -->
-    <div class="z-10 bottom-0 w-full bg-white border border-solid border-gray-300 rounded-t-md shadow-upward">
+    <div class="fixed z-10 bottom-0 w-full bg-white border border-solid border-gray-300 rounded-t-md shadow-upward sm:relative">
```

Comment:
Can we also use `flex` to make the player fixed at bottom on mobile screens, similar to how we achieved this on desktops? That would be preferred.